source/Libraries/Iciclecreek.Bot.Builder.Dialogs.Recognizers.Lucene/Templates/en-us/form-operations.en-us.data

```diff
@@ -0,0 +1,63 @@
+>${comment?} The >> is to mark this as the source of entities when doing cross entity examples
+
+>> entity: Standalone entities that map to multiple properties${simple?}
+- ${simple*}
+
+>${comment} We ensure that each value occurs at least once here to support reliably resolving entity ambiguity
+>> entity: Standalone ${property} with ${entity}
+- {@${property}=${unique*}}
+- {@${property}=${uniqueOrValue*} ${name}}
+
+>> property: Standalone ${property} name
+- {@${property}=${name}}
+
+> Add() examples for ${property} and ${entity}
+- {@Add()=add {@${property}=${uniqueOrValue} ${name}}}
```

Comment:
I don't think you need to include this whole file?  Can't you get it from the standard templates?